### PR TITLE
changelog and Cromwell versions

### PR DESCRIPTION
**Pull request by denis-yuen**
noticed typo

### CODE REVIEW

File: _docs/advanced-features.md [hunk 1]
```diff
@@ -163,10 +163,10 @@ For remote launches, we download the zip directory as returned by the Dockstore
 
 ## WDL Launcher Configuration
 
-By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 30.2.
+By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 36.
 Additionally, you can override the cromwell version in your `~/.dockstore/config` using:
 ```
-cromwell-version = 34
+cromwell-version = 36
```
Comment (garyluu) [inline _docs/advanced-features.md]: I think the override was supposed to be different than the default

**Comment by denis-yuen** [comment]
For reviewers, suggest anything that you want highlighted in the changelog

**Comment by denis-yuen** [comment]
> uploading hosted workflow via zip?

I debated this one, I think I want to get it hooked up and/or tested by the HCA before advertising it.

File: _posts/2019-04-08-Dockstore-1.6.0.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  * select between classic and [EPAM](https://github.com/epam/pipeline-builder) visualizations for WDL workflows
+  * improved support for Nextflow workflows
+      * using Nextflow code to parse and process workflow authors and descriptions for better compatibility
+* beta support for launching workflows directly onto GA4GH WES endpoints
+  * see our first steps at [WES CLI](https://docs.dockstore.org/docs/publisher-tutorials/advanced-features/)
+* launch-with support for [Terra](https://terra.bio/)
+* improved display of validation errors
+* support for [aliases](https://docs.dockstore.org/docs/publisher-tutorials/guid-alias/) for advanced developers
+* a huge host of bugfixes for everything from Nextflow DAGs, our support for GA4GH TRS endpoints, the usability of search, description file encoding, display of files on iOS, and display of error messages
+
```
Comment (coverbeck) [inline _posts/2019-04-08-Dockstore-1.6.0.markdown]: Do you want to mention support for gs file provisioning plugin?

File: _docs/advanced-features.md [hunk 3]
```diff
@@ -163,10 +163,10 @@ For remote launches, we download the zip directory as returned by the Dockstore
 
 ## WDL Launcher Configuration
 
-By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 30.2.
+By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 36.
```
Comment (wshands) [inline _docs/advanced-features.md]: Shouldn't this say "By default, WDL tools/workflows will automatically be **run** with [cromwell](https://github.com/broadinstitute/cromwell) 36."

**Pull request merged**